src/benchmarks/micro/corefx/System.Threading/Perf.Thread.cs

```diff
@@ -12,5 +12,13 @@ public class Perf_Thread
     {
         [Benchmark]
         public Thread CurrentThread() => Thread.CurrentThread;
+
+#if !NETFRAMEWORK
+        [BenchmarkCategory(Categories.CoreFX)]
```

Comment:
The same attribute is on the type already. Does it need to be repeated here again?

Reply:
Not after i moved it here from  LowLevel\Perf.cs where i put it originally.